app/src/main/java/test1/daihocbachkhoa/tranphuc1995/assignment1/MainActivity.java

```diff
@@ -0,0 +1,91 @@
+    private StringBuilder orderBeverages = new StringBuilder();
+    private EditText edtNumberOfOrder;
+    private ArrayList<String> arlNumberOfOrder = new ArrayList<>();
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        createData();
+        init();
+        confirmOrder();
+    }
+
+    private void init(){
+        recyclerView = (RecyclerView)findViewById(R.id.recyclerViewList);
+        btnConfirmOrder = (Button)findViewById(R.id.button_Confirm);
+        edtNumberOfOrder = (EditText)findViewById(R.id.editText_NumberOfOrder);
+        arlDrink = createData();
+        adapter = new RecyclerViewAdapter(arlDrink);
+        LinearLayoutManager layoutManager = new LinearLayoutManager(getApplicationContext());
+        layoutManager.setOrientation(LinearLayoutManager.VERTICAL);
+        recyclerView.setLayoutManager(layoutManager);
+        recyclerView.setAdapter(adapter);
+
+    }
+
+    private ArrayList<Beverage> createData(){
+        ArrayList<Beverage> arlData = new ArrayList<Beverage>();
+        Beverage coffee =  new Beverage("Coffee",R.drawable.coffee);
+        arlData.add(coffee);
+        Beverage orange = new Beverage("Pure Orangle Juice",R.drawable.orange);
+        arlData.add(orange);
+        Beverage carrot =  new Beverage("Carrot Juice", R.drawable.carrot);
+        arlData.add(carrot);
+        Beverage waterMelon =  new Beverage("Water Melon Juice", R.drawable.watermelon);
+        arlData.add(waterMelon);
+        Beverage guava =  new Beverage("Guava Juice", R.drawable.guava);
+        arlData.add(guava);
+        Beverage heniken =  new Beverage("Heineken", R.drawable.heineken);
+        arlData.add(heniken);
+        return arlData;
+    }
+
+    private void confirmOrder(){
+        btnConfirmOrder.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                orderBeverages.setLength(0);
+                for(int i=0 ; i< adapter.arlCheckBeverages.size() ; i ++){
+                    orderBeverages.append(adapter.arlCheckBeverages.get(i).getCheckBeverages()+ "  x");
+                    orderBeverages.append(adapter.arlCheckBeverages.get(i).getNumberOfOrder());
+                    orderBeverages.append("\n");
+                }
+                if(orderBeverages.length() >0){
+                    orderBeverages.setLength(orderBeverages.length() -1);
+                }
+                if(adapter.arlCheckBeverages.size() > 0){
+                    showBeverages(orderBeverages.toString());
+                }
+                else{
+                    Toast.makeText(MainActivity.this, "Please order Beverage !", Toast.LENGTH_LONG).show();
+                }
+            }
+        });
+    }
+
+    private void showBeverages(String listOrder){
+        Intent it = new Intent(MainActivity.this,ShowBeverages.class);
```

Comment:
```
Intent it = new Intent(MainActivity.this,ShowBeverages.class);
```
Bad format this line of code.

Better format:
```
Intent it = new Intent(MainActivity.this, ShowBeverages.class);
```
It's focusing on details that makes a developer progress in his field.
**Tips:**   *There is auto-format functionality of our awesome Android Studio, use that as a practice when do coding.*